package.json

```diff
@@ -11,10 +11,8 @@
 		"url": "https://sindresorhus.com"
 	},
 	"type": "module",
-	"exports": {
-		"types": "./dist/index.d.ts",
-		"default": "./dist/index.js"
-	},
+	"exports": "./dist/index.js",
+	"types": "./dist/index.d.ts",
```

Comment:
Don't make unrelated changes.

Reply:
Unrelated, but current packaging is broken. This puts back to the previous working version, otherwise I couldn't npm install from a git branch.

Reply:
The existing config is totally valid, so I think something else is going on for you.

Reply:
Also, if you make unrelated changes, at least comment on why you're doing it. Don't wait for a reviewer to point it out. That just wastes time on both sides.